frontend/src/pages/modelRegistry/screens/components/DeployRegisteredModelModal.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { Button, Form, Modal } from '@patternfly/react-core';
+import { ModelVersion } from '~/concepts/modelRegistry/types';
+import { ProjectKind } from '~/k8sTypes';
+import useProjectErrorForRegisteredModel from '~/pages/modelRegistry/screens/RegisteredModels/useProjectErrorForRegisteredModel';
+import ProjectSelector from '~/pages/modelServing/screens/projects/InferenceServiceModal/ProjectSelector';
+import { ManageKServeModalWrapper } from '~/pages/modelServing/screens/projects/kServeModal/ManageKServeModal';
+import useServingPlatformStatuses from '~/pages/modelServing/useServingPlatformStatuses';
+import { getProjectModelServingPlatform } from '~/pages/modelServing/screens/projects/utils';
```

Comment:
There's also a bunch of stuff being imported across `~/pages/modelServing` and `~/pages/modelRegistry` here - maybe we should take a pass and see if any of it is a good candidate to move to `~/concepts/modelServing`....

...ahh, we don't yet have a `~/concepts/modelServing`. Maybe we should open a followup tech tebt issue to create it and lift some of these things out there after MR MVP. I may be stirring up too much refactoring here with these comments 😄.

Yeah ok, for stuff you're importing from `~/pages/modelServing` let's leave them alone for now and I'll open a followup issue, but for new stuff you're introducing we should consider putting stuff in `~/concepts/modelRegistry` if we can do so cleanly. I'll leave it up to your judgement how much of that is within the scope of this PR.

Reply:
Yeah I agree, I will leave this PR as it is for now because I checked the usage in those hooks, and it seems it's cross-page using many enums/const so I think they should be restructured afterward to lift all of them up to the `/concepts` folder in another ticket.